ts/a11y/semantic-enrich.ts

```diff
@@ -244,12 +244,21 @@ export function EnrichedMathItemMixin<N, T, D, B extends Constructor<AbstractMat
      */
     public attachSpeech(document: MathDocument<N, T, D>) {
       if (this.state() >= STATE.ATTACHSPEECH) return;
+      if (this.isEscaped || !document.options.enableEnrichment) {
+        this.state(STATE.ATTACHSPEECH);
+        return;
+      }
       let [speech, braille] = this.existingSpeech();
       let [newSpeech, newBraille] = ['', ''];
       if (!speech || !braille ||
         document.options.enableSpeech || document.options.enableBraille) {
```

Comment:
I didn't catch this earlier, but I think this condition is wrong.  That is, if `document.options.enableSpeech` is false, you should not generate speech even if `!speech` is true, and similarly for braille.  So I think this should be

``` js
if ((!speech && document.options.enableSpeech) || (!bralle && document.options.enableBraille)) {
```

Reply:
Thanks. Good catch.